tests/evaluation_metrics/conftest.py

```diff
@@ -1,10 +1,12 @@
+"""conftest."""
 import pytest
 
 from datasetinsights.data.bbox import BBox2D
 
 
 @pytest.fixture
 def get_gt_pred_bbox():
+    """get gt pred bbox."""
```

Comment:
Python docstrings for unittests are usually nice to have. Why go through every tests and adding docstings does not add helpful info (as this is simply the test function name)?

Reply:
Its pep257 linting requirement